skeleton/aws/main.tf

```
@@ -0,0 +1,13 @@
+terraform {
+  cloud {
+    organization = var.terraform_organization
+
+    workspaces {
+      name = var.terraform_workspace
+    }
+  }
+}
+
+provider "aws" {
+  region = var.region
+}
```

please consider adding the default tags in the main file so we don't need to define it in every module
```suggestion
provider "aws" {
  region = var.region
  
  default_tags {
    tags = {
      Environment = var.environment
      Owner       = var.owner
    }
  }
}
```

we started doing that in worpt infra https://github.com/worpt-wallet/wallet-infra/blob/de23f4faed49ffdd2975eac2f5e87bba254d90f6/staging/main.tf#L14-L19

---

Added in https://github.com/nimblehq/infrastructure-templates/pull/26/commits/f05d07f87b09f0f40857a25b932a4384cfa6e69a